packages/sdk/src/types/blockItemSummary.ts

```diff
@@ -514,7 +520,7 @@ export function affectedAccounts(summary: BlockItemSummary): AccountAddress.Type
         case TransactionKindString.TransferToPublic:
             return [summary.removed.account];
         case TransactionKindString.Update: {
-            return summary.events.reduce(
+            return summary.events.filter(isKnown).reduce(
```

Comment:
Same:
https://github.com/Concordium/concordium-rust-sdk/blob/forward-compatible/src/types/mod.rs#L1922

Note: By the way, the `rust-SDK` includes addresses that their `PLT` balance in the return value now.

Reply:
Good catch, fixed this 😄